### DISCUSSION TITLE
Coste anual por mercado de Kw (€)

### DISCUSSION THREAD

**Discussion by JaviElio** [uvejota/homeassistant-edata#196] [Q&A] [answered]
Buenas,
Quiero activar la opción de facturación. Entiendo bien todos los parámetros que hay que introducir menos el "Coste anual por mercado de Kw (€)".
¿Este coste donde se puede obtener? ¿En la factura? En la mia no lo veo reflejado.
Muchas gracias.

**Comment by alfonsocasadodiez** [comment, marked as answer]
3.113
[https://selectra.es/energia/info/que-es/pvpc](url)

**Comment by cuchipandy** [comment]
Segun entiendo ese coste solo seria aplicable a la tarifa PVPC pero los que tenemos tarifa comercial no tiene sentido, no?

**Comment by JaviElio** [comment]
Eso entiendo también. No tengo PVPC y ese coste lo tengo a 0.001 ya que la integración no permite dejarlo a cero.

**Comment by uvejota** [comment]
Esto ha sido eliminado en la última versión de edata al seleccionar una tarifa "custom".
